recipes-qt/qt5/qtwebengine_git.bb

```diff
@@ -197,8 +197,8 @@ SRC_URI_append_libc-musl = "\
     file://chromium/0029-chromium-musl-initialize-msghdr-in-a-compatible-mann.patch;patchdir=src/3rdparty \
 "
 
-SRCREV_qtwebengine = "f5268555099a1a96f8730035a40f04a473dd396e"
-SRCREV_chromium = "7b2f027ea83c372c33d5b50deb65a2d98244aa04"
+SRCREV_qtwebengine = "5537ff4437ea7a5f9ea140071343f88bf48deddc"
+SRCREV_chromium = "fb6ab5e483876298235be1c6a6013b426c82b759"
```

Comment:
Could you please update this to the latest available commit in the qtwebengine-chromium repo (f5a93d251cca15cb24a3acacd2fc0f645513c56d), otherwise it's missing a fix for embedded builds which activate printing-and-pdf.

Reply:
Thank you your feedback. I will check this.